crates/network-scanner/src/scanner.rs

```diff
@@ -351,3 +349,39 @@ pub struct NetworkScannerParams {
     pub mdns_single_query_timeout: u64, // in milliseconds
     pub max_wait_time: u64,             // max_wait for entire scan duration in milliseconds, suggested!
 }
+
+#[derive(Debug, Clone)]
+pub enum Protocol {
+    /// Wayk Remote Desktop Protocol
+    Wayk,
```

Comment:
suggestion: I think you can safely remove Wayk from this list too. The associated product is dead.

Reply:
updated, removed non-standard protocols